src/app/company/company-add/company-add.component.css

```diff
@@ -0,0 +1,28 @@
+.company-main-box {
```

Comment:
The box is stuck at the top of the page, it needs some padding, or if you find another solution, that's even better.

Reply:
Fixed, but used a box shadow instead of a border. Let me know if your happy with the changes.

Reply:
Looks awesome, made my day :D